ui/app/components/job-page/abstract.js

```diff
@@ -1,9 +1,7 @@
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
-import classic from 'ember-classic-decorator';
 
-@classic
```

Comment:
i haven't looked at this in depth yet, but  🔜

dropping these in for reference (for myself) https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/ and https://guides.emberjs.com/release/upgrading/current-edition/#toc_octane-upgrade-strategy